.github/workflows/test-unit-tests.yml

```diff
@@ -1,4 +1,4 @@
-name: Unit tests
+name: Test Unit tests
```

Comment:
That sounds weird / repetitive?

Reply:
Yes, technically "Test units" would be enough - but that sounded even weirder.
And "Run unit tests" would not fit the pattern that I am trying to have in all the job names 🤷 
Suggestions?

<img width="449" alt="image" src="https://github.com/prisma/prisma-engines/assets/183673/55648e84-372a-4a08-ac85-0084dc7e123c">

Reply:
Well, what you suggested is fine I think